docs/sql-ref-ansi-compliance.md

```diff
@@ -384,6 +384,8 @@ When ANSI mode is on, it throws exceptions for invalid operations. You can use t
   - `try_make_timestamp_ltz`: identical to the function `make_timestamp_ltz`, except that it returns `NULL` result instead of throwing an exception on error.
   - `try_make_timestamp_ntz`: identical to the function `make_timestamp_ntz`, except that it returns `NULL` result instead of throwing an exception on error.
   - `try_make_interval`: identical to the function `make_interval`, except that it returns `NULL` result instead of throwing an exception on invalid interval.
+  - `try_elt`: identical to the function `elt`, except that it returns `NULL` result instead of throwing an exception on index out of bounds.
```

Comment:
where is `try_elt` defined?

Reply:
You are right, I was thinking about whether we need this one, but it turns out we can use `get` + `explode` as a workaround.